lage.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
       inputs: ['*', 'src/**/*', 'assets/**/*'],
       outputs: ['lib/**/*', 'lib-commonjs/**/*'],
     },
-    buildci: ['build', 'test', 'lint', 'depcheck'],
+    buildci: ['build', 'test', 'lint', 'depcheck', 'check-publishing'],
```

Comment:
Can we add `lint-lock file` and `align-deps` here too? I couldn't because those run on the root and need the root to be a workspace, which then required us to specify inputs/outputs. Maybe now, this is better,

Reply:
Added a new command "pr-check" that does everything. Because this command is referenced in the pipeline tasks I didn't want to change that now.